src/Components/Views/Recommendations/List/list.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styled from 'react-emotion'
+import { RecommendationItem } from '../Item/item'
+
+export const ListWrapper = styled('ul')`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+`
+
+export class RecommendationList extends React.Component {
+  render() {
+    const { items } = this.props
+    return items.length ? (
+      <ListWrapper>
+        {items.map(item => (
+          <RecommendationItem key={item.item_id} item={item} />
```

Comment:
should this key be `item.fee_item_id` and not `item.item_id` @collectedmind 

<img width="835" alt="screen shot 2018-07-16 at 1 21 30 pm" src="https://user-images.githubusercontent.com/1546207/42781622-7981a56e-88fb-11e8-9db6-2dc0955fe9b0.png">

<img width="396" alt="screen shot 2018-07-16 at 1 21 23 pm" src="https://user-images.githubusercontent.com/1546207/42781623-79967ca0-88fb-11e8-8df2-8b5efa15bb20.png">

Reply:
👍